images/Images/ImageConcat.tcc

```diff
@@ -259,9 +259,11 @@ Bool ImageConcat<T>::setImageInfo (const ImageInfo& info)
   for (uInt i=0; i<latticeConcat_p.nlattices(); ++i) {
     ImageInterface<T>& img =
       dynamic_cast<ImageInterface<T>&>(*(latticeConcat_p.lattice(i)));
-    ndone += img.rwImageInfo().setInfoSplitBeamSet (ndone, info, img.shape(), 
+    ImageInfo& rwii = img.rwImageInfo();
```

Comment:
I think it is better to do
    ImageInfo rwii;
which works fine. It is somewhat dirty to change the internal ImageInfo object and set it to itself.

Reply:
ok, then there is no point in getting the rw version. one can just make a copy of the read-only version if it is going to be copied anyway.